src/remote.ts

```diff
@@ -55,8 +49,45 @@ export class Remote {
     if (parts.length < 2 || parts.length > 3) {
```

Comment:
Is there a reason why this isn't `parts.length !== 2`?

Reply:
Woah I did not even notice that.  No idea lol, someone else wrote this

It does seem weird.  I might as well change it while I am in here!

Reply:
Oh wait it is `2` or `3`.  Still seems better to check the range we want rather than the range we do not want though.

Reply:
Oh, wait, yeah, you're right. Guess I was tired, and forgot that it would be inclusive for [2, 3]